texts/smlm-lab-course/smlm-lab-manual.tex

```diff
@@ -368,6 +368,25 @@ \subsection{Photoswitching vs Photoactivation}
     \label{table:photophysics-comparison}
 \end{table}
 
+\section{The Role of the Buffer}
+
+The STORM buffer used in this course contains the following components.
+
+\begin{itemize}
+    \item Trizma base
+    \item NaCl
+    \item Glucose
+    \item Monoethanolamine (commonly called MEA)
+    \item Glucose oxidase
+    \item Catalase diluted in glycerol
+\end{itemize}
+
+These components serve three broad purposes. The first is to maintain the pH of the buffer at a constant value. Different fluorescent dyes perform better at different pH values, and there does not seem to be an easily discernible trend to predict the best pH for a given dye. The pH of each buffer must therefore be tuned to a specific dye through experimentation. Trizma and NaCl together help maintain the correct pH of the buffer.
```

Comment:
Instead of "Different fluorescent dyes perform better at different pH values" you could be more direct by saying "the photoswitching behavior of a given fluorophore is effected by pH and there is not yet a known rule to predict how pH influences this behavior between different fluorophore species."